scripts/scripts.js

```diff
@@ -197,6 +197,64 @@ export const fetchLangDatabyFileName = async (fileName) => {
   }
 };
 
+/**
+ * Create section background image
+ *
+ * @param {*} doc
+ */
+function decorateSectionImages(doc) {
+  const sectionImgContainers = doc.querySelectorAll('main .section[data-image]');
+  sectionImgContainers.forEach((sectionImgContainer) => {
+    const sectionImg = sectionImgContainer.dataset.image;
+    const sectionTabImg = sectionImgContainer.dataset.tabImage;
+    const sectionMobImg = sectionImgContainer.dataset.mobImage;
+    let defaultImgUrl = null;
+
+    const picture = document.createElement('picture');
+    if (sectionImg) {
+      const sourceDesktop = document.createElement('source');
+      const { pathname } = new URL(sectionImg, window.location.href);
+      sourceDesktop.type = 'image/webp';
+      sourceDesktop.srcset = `${pathname}?width=1920&format=webply&optimize=medium`;
+      sourceDesktop.media = '(min-width: 1024px)';
+      picture.appendChild(sourceDesktop);
+      defaultImgUrl = pathname;
+    }
+
+    if (sectionTabImg) {
+      const sourceTab = document.createElement('source');
+      const { pathname } = new URL(sectionTabImg, window.location.href);
+      sourceTab.type = 'image/webp';
```

Comment:
can these lines of code to be moved to common function with source as one of the argument for differentiation

Reply:
done